react/src/components/datavis/DataTable/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import ReactTable from 'react-table';
+
+import '../../../../node_modules/react-table/react-table.css';
```

Comment:
This shouldn't use a relative path. You should be able to do 'react-table/react-table.css';